### ISSUE TITLE
Ambassador application

### ISSUE THREAD

**Issue by benjagm** [json-schema-org/TSC#21]
Dear TSC, 

I'd like to ask for your vote on this JSON Schema Ambassador application.  They have been creating good articles and other senior ambassadors are ok with accepting them. They have my support and I'd like to start the voting process to get your feedback.

https://github.com/json-schema-org/community/pull/823

**Comment by gregsdennis** [comment]
As I stated in the PR, I feel that being an ambassador requires a certain level of direct involvement in the community, which I haven't seen from him.

He joined the slack workspace in Sept and posted a couple of his articles for us to review.  He's been quiet since.

Other ambassadors are actively engaged with the community, and I'd like to see this in an applicant before I approve this role for them.

**Comment by mwadams** [comment]
I think we should be encouraging (and welcome the contribution) but I agree it feels a bit early.

**Comment by jviotti** [comment]
I agree with other comments. I never interacted with this person before in any way

**Comment by jdesrosiers** [comment]
I have to disagree with the general consensus of the responses so far, but I'll start with the parts I do agree with. I agree that we haven't seen much of this person in the community yet. Searching in Slack, I find his first post from September 19th. That means he's only been active in this community for about 2 months. That seems a short time to consider someone an ambassador for our brand.

That said, if you search his handle in Slack, you'll see he's making an effort to be involved in the community. But, apart from @gregsdennis, we haven't really engaged with him despite him writing decent articles. He posts about his articles and asks for feedback. Sometimes he gets engagement and sometimes he doesn't. I don't think we can fault people for not being active enough when we aren't engaging when they reach out.

Finally, there is precedent (https://github.com/json-schema-org/community/pull/732) for accepting ambassadors with less community engagement and similar time in the community as this applicant. We could say that was a mistake and we want to raise the standards going forward, but I would prefer to be more accepting for initial applicants.

I don't remember exactly, but didn't we say that ambassador positions will expire after a year and they need to be renewed? It's not a lifetime appointment, right? I say that we should admit people like the applicant who are showing effort and interest, but when time comes to renew, we hold them to a higher standard. If they don't continue to contribute to the growth of the community since their appointment, they will be removed. Maybe we even call these "junior ambassadors" or "provisional ambassadors" until they get past their first renewal.

Given precedent, the effort the applicant has shown, and the impermanence of the position, I'm voting in favor of accepting the applicant. However, I would want to see more that just a couple articles for renewal. I'd want to see them become a recognizable figure of the community.

**Comment by gregsdennis** [comment]
Jason makes some good points, and I'm willing to give him a "first term" trial.  I've changed my vote.  @mwadams @jviotti, please read and reassess.

**Comment by mwadams** [comment]
That's persuasive. We need to make sure everyone is clear when and how we are doing "annual renewals" to ensure we are fair to everyone with respect to this kind of "see how it goes".

**Comment by jviotti** [comment]
Sounds good to me! Let's give him a shot

**Comment by Relequestual** [comment]
I'm actually not in favour at this point in time.

I've read two of the articles linked so far, and, there are some serious issues.
Both articles mention ajv's strict mode, but both of them (using the same phrasing) say the opposite of what's correct. I appreciate this could be a language barrier issue.

One of the articles, titled "Extending JSON Schema with Custom Vocabulary", doesn't actually do this at all. What it does do is show how to add custom key words in ajv, and uses draft-07.

This feels like a drive by attempt to become an ambassador, personally.

While I totally agree we want to encourage applications, and while we have accepted those with less content and engagment, I think those instances haven't included articles which have a few things plain wrong or not what the title suggests. IMHO those articles are actually detrimental , and I don't say that lightly. (I've seen other similar articles linked by people asking questions, pointing to the article. I have to explain, the article is wrong, sorry).

Simply, I cannot endorse someone when the content they have written is not just a little wrong, but flat out opposite of what's true, and results in what I feel is a net negative.

If they want to get feedback on the articles and correct them, then we can discuss again.

**Comment by jdesrosiers** [comment]
> Both articles mention ajv's strict mode, but both of them (using the same phrasing) say the opposite of what's correct.

Sorry, this is largely my fault. I accidentally told him the wrong thing. You can see the discussion starting [here](https://github.com/json-schema-org/community/pull/823#issuecomment-2507209096).

**Comment by Relequestual** [comment]
> > Both articles mention ajv's strict mode, but both of them (using the same phrasing) say the opposite of what's correct.
> 
> Sorry, this is largely my fault. I accidentally told him the wrong thing. You can see the discussion starting [here](https://github.com/json-schema-org/community/pull/823#issuecomment-2507209096).

Understood, although I'm not sure it was what you said, unless something has since been removed.

I would object less if that part of the articles gets fixed.

The talk they are giving in Jan 2025 is a positive indication. It looks information dense too.

**Comment by benjagm** [comment]
Hi Team, I think we have a majority in positive votes. Shall we proceed with accepting and announcing them?

**Comment by benjagm** [comment]
Finally this proposal has been rejected.  Closing it as finished.
